### DISCUSSION TITLE
How can I have the same snippet be triggered by multiple strings?

### DISCUSSION THREAD

**Discussion by danielo515** [L3MON4D3/LuaSnip#711] [Q&A]
Sometimes I have a snippet that I want to be able to trigger using several strings. Mostly mnemonics, sometimes I remember one word better than another. Is it possible to pass a list of words as the trigger? If not, what would be the best alternative?

**Comment by L3MON4D3** [comment]
See #420 :P

**Reply by danielo515** [reply]
Thanks for the pointer. I made a quesiton on that thread you may want to ask:
> When you talk about deep copy, do you mean just vim.deepcopy or is the copy functionality luasnip has?
